### PR TITLE
Add LoginView component

### PR DESCRIPTION
**Pull request by Shadic78**
Co-authored-by: Raul-Morales <r20160106@gmail.com>

# Description
What did you implemented/Why did you implemented this:
 - Created the LoginView component without functionality.
  
  ## Screenshots 
image

### CODE REVIEW

**Review by YilKanda — APPROVED** [review]
LGTM!

File: src/screens/LoginView/LoginView.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <Text style={styles.title}>Sign In</Text>
+      <View style={{marginBottom: 10}}>
+        <Text style={styles.textInputLabel}>Email</Text>
+        <TextBox style={styles.textInputLabel} value={email} onChange={setEmail} />
+      </View>
+      <View>
+        <Text style={styles.textInputLabel}>Password</Text>
+        <TextBoxWithButton style={styles.textInputLabel} type='password' value={password} onChange={setPassword}  />
+      </View>
+      <View style={{marginTop: 30, marginBottom: 10}}>
```
Comment (YilKanda) [inline src/screens/LoginView/LoginView.jsx:23]: Do not add styles directly, it is better to import them from a styles file.

File: src/screens/LoginView/LoginView.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import {styles} from '../RegisterView/RegisterViewStyle';
+
+const LoginView = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  return(
+    <ScrollView style={styles.container}>
+      <Text style={styles.title}>Sign In</Text>
+      <View style={{marginBottom: 10}}>
```
Comment (YilKanda) [inline src/screens/LoginView/LoginView.jsx:15]: Do not add styles directly, it is better to import them from a styles file.

**Review by YilKanda — CHANGES_REQUESTED** [review]

File: src/screens/LoginView/LoginView.jsx (hunk 1 above)
Comment (Shadic78) [reply src/screens/LoginView/LoginView.jsx]: Done.

**Review by YilKanda — APPROVED** [review]
LGTM

**Pull request merged**